source/includes/kubernetes_extension/_k8_charts.md

```diff
@@ -87,28 +60,30 @@ Required | &nbsp;
-`attributes.repoUrl` <br/>*string* | The url of the source of the chart.
-`attributes.description` <br/>*string* | The chart description.
-`attributes.home` <br/>*string* | The home url for the chart.
-`attributes.keywords` <br/>*string* | The keywords linked to the chart.
-`attributes.maintainers` <br/>*object* | The maintainers of the charts and their information.
-`attributes.sources` <br/>*list* | The sources for the chart.
-`attributes.icon` <br/>*string* | The icon for the chart.
-`relationships` <br/>*objection* | The relationship information for the chart.
-`relationships.latestChartVersion` <br/>*object* | The lastest version chart information.
-`relationships.latestChartVersion.data.version` <br/>*string* | The digest validating the chart.
-`relationships.latestChartVersion.data.digest` <br/>*string* | The name of the chart.
-`relationships.latestChartVersion.data.urls` <br/>*string* | The URLs for the package to download.
-`relationships.latestChartVersion.data.readme` <br/>*string* | The readme file location.
-`relationships.latestChartVersion.data.values` <br/>*string* | The values file location.
-`version`<br/>*string* | The revision of the release.
-`insecureServer`<br/>*boolean* | Specify if to access the chart we need an unsecure connection.
-
+| Attributes                                         | &nbsp;                                                                                                     |
+|----------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| `id` <br/>_string_                                 | The id for the chart, of the form `repositoryName/chartName`.                                              |
+| `name` <br/>_st                                    | The chart name.                                                                                            |
+| `description` <br/>_string_                        | The chart description.                                                                                     |
+| `version` <br/>_string_                            | The latest chart version.                                                                                  |
+| `license` <br/>_string_                            | The chart's license.                                                                                       |
+| `deprecated` <br/>_boolean_                        | True if the chart is deprecated.                                                                           |
+| `repository` <br/>_object_                         | The chart's parent repository.                                                                             |
+| `repoository.url` <br/>_string_                    | The repository url.                                                                                        |
```

Comment:
isn't URL in caps?

Reply:
Fixed here https://github.com/cloudops/cloudmc-api-docs/pull/261/commits/498b4777ba7f890f506f25c754059b8484c23de2#diff-3cbb097edb0517bba241163d615649f31d6e6cf393ad6356b544586e5d1ff8e8R168